src/theme-preval.js

```diff
@@ -58,35 +59,73 @@ const darkDimmed = partitionColors(primitives.colors['dark_dimmed'])
 // so we need to use JSDoc comments.
 
 /**
- * @type Partial<typeof primitives.colors.light>
+ * @type {Omit<typeof light['colors'], 'scale'>}
  */
 const lightColors = omitScale(light.colors)
 
 /**
- * @type Partial<typeof primitives.colors.light>
+ * @type {Omit<typeof light['shadows'], 'scale'>}
  */
 const lightShadows = omitScale(light.shadows)
 
 /**
- * @type Partial<typeof primitives.colors.dark>
+ * @type {Omit<typeof dark['colors'], 'scale'>}
  */
 const darkColors = omitScale(dark.colors)
 
 /**
- * @type Partial<typeof primitives.colors.dark>
+ * @type {Omit<typeof dark['shadows'], 'scale'>}
  */
 const darkShadows = omitScale(dark.shadows)
 
 /**
- * @type Partial<typeof primitives.colors.dark_dimmed>
+ * @type {Omit<typeof darkDimmed['colors'], 'scale'>}
  */
 const darkDimmedColors = omitScale(darkDimmed.colors)
 
 /**
- * @type Partial<typeof primitives.colors.dark_dimmed>
+ * @type {Omit<typeof darkDimmed['shadows'], 'scale'>}
  */
 const darkDimmedShadows = omitScale(darkDimmed.shadows)
 
+/**
+ * @todo are these types all correct - especially the partial colors from
+ * primitives?  if not - let's tighten that up!
+ *
+ * @typedef {Object} ThemePrevalType
```

Comment:
It would be awesome if we just had this - 

It might be valuable for me to just take the theme object from primer.style theme reference and generate types directly from that via a json -> type converter (like I did for shadows/colors paritionig), and just import('....') that type here instead of defining it in this way